Dependencies/xr/Source/ARKit/XR.mm

```diff
@@ -933,6 +1024,113 @@ void UpdatePlanes(std::vector<Frame::Plane::Identifier>& updatedPlanes, std::vec
             }
         }
 
+        /**
+         Updates existing meshes in place, gets the list of updated/created mesh IDs and removed mesh IDs.
+         */
+        void UpdateMeshes(std::vector<Frame::Mesh::Identifier>& updatedMeshes, std::vector<Frame::Mesh::Identifier>& deletedMeshes) {
+            if (![sessionDelegate TrySetMeshDetectorEnabled: true]) {
+                return;
+            }
+
+            if (@available(iOS 13.4, *)) {
+                [sessionDelegate LockMeshes];
+                @try {
+                    // Update all meshes that have been updated since the last frame
+                    auto updatedARKitMeshes = [sessionDelegate GetUpdatedMeshes];
+                    for (ARMeshAnchor* updatedMesh : *updatedARKitMeshes) {
+                        const auto& geometry = updatedMesh.geometry;
```

Comment:
I think this file uses uniform initialization everywhere else, so it would be ideal to make this new code consistent. For this line for example, it would be:
```suggestion
                        const auto& geometry{ updatedMesh.geometry };
```
(and that would be applied everywhere)